master/internal/prom/prom.go

```diff
@@ -104,21 +104,33 @@ func AssociateContainerRuntimeID(cID cproto.ID, dcID string) {
 	containerIDToRuntimeID.WithLabelValues(dcID, cID.String()).Inc()
 }
 
-// AddAllocationReservation associates allocation and container and container and GPUs.
-func AddAllocationReservation(summary sproto.ContainerSummary,
-	containerStarted *sproto.TaskContainerStarted) {
-	AssociateAllocationContainer(summary.AllocationID, summary.ID)
-	AssociateContainerRuntimeID(summary.ID, containerStarted.NativeReservationID)
-	for _, d := range summary.Devices {
-		AssociateContainerGPU(summary.ID, d)
+// AddAllocationResources associates allocation and container and container and GPUs.
+func AddAllocationResources(summary sproto.ResourcesSummary,
+	containerStarted *sproto.ResourcesStarted) {
+	if summary.ContainerID == nil {
```

Comment:
this is mostly a todo to figure out for slurm, but will just be a noop until it is thought out.